core/email_api.php

```diff
@@ -538,11 +538,18 @@ function email_signup( $p_user_id, $p_confirm_hash, $p_admin_name = '' ) {
  * @return void
  */
 function email_send_confirm_hash_url( $p_user_id, $p_confirm_hash ) {
-	if( OFF == config_get( 'send_reset_password' ) ||
-		OFF == config_get( 'enable_email_notification' ) ) {
+	if( OFF == config_get( 'send_reset_password' ) ) {
+		log_event( LOG_EMAIL_VERBOSE, 'Password reset email notifications disabled.' );
```

Comment:
LOG_EMAIL_VERBOSE used like in this case:
https://github.com/mantisbt/mantisbt/blob/master/core/email_api.php#L1338

For LOG_EMAIL_RECIPIENT, can be changed to LOG_EMAIL, if you think it fits better.